### ISSUE TITLE
Unexpected error: <class 'selenium.common.exceptions.WebDriverException'>

### ISSUE THREAD

**Issue by hoyeunglee** [SeleniumHQ/selenium#4768]
Selenium version 3.3.3

i used selenium a long time and can run without errors,
but today it suddenly got this error when click a link of web at footer of openstack management web
in chrome Version 61.0.3163.100 (Official Build) (64-bit)

```
selenium.find_element_by_css_selector("button.btn").click()
selenium.implicitly_wait(5)

selenium.find_element_by_link_text("Instances").click();
selenium.implicitly_wait(7)
numberofpages = 1
numberofmore = 1

while numberofmore > 0:
    #print("here1")
    table_id = selenium.find_element_by_id("instances")
    #print("here2")
    rows = table_id.find_elements(By.TAG_NAME, "tr") # get all of the rows in the table
    #print("here3")
    totalofstates = 0
    counterofrunning = 0
    for row in rows:
        totalofstates = totalofstates + 1
        try: 
            col = row.find_elements(By.TAG_NAME, "td")
            #print("here4")
            if len(col) > 5:
                #print(col[1].text)
                #print(col[9].text)
                if col[9].text == "Running":
                    counterofrunning = counterofrunning + 1
        except:
            print("one exception")
    successrate = counterofrunning/totalofstates
    if successrate > 0.7:
        print("instance page "+str(numberofpages)+" ok")
    else:
        print("instance page "+str(numberofpages)+" need investigate")
    #print("here6")
    numberofmore = 0
    tfoots = table_id.find_elements(By.TAG_NAME, "tfoot")
    #print("here7")
    for tfoot in tfoots:
        rows = tfoot.find_elements(By.TAG_NAME, "tr")
        for row in rows:
            col = row.find_elements(By.TAG_NAME, "td")
            #print("here8")
            numberofmore = len(col)
            #print("numberofmore=", numberofmore)
            if numberofmore > 0:
                try:
                    collink = col[0].find_elements(By.TAG_NAME, "a")
                    print("numberoflink=", len(collink))
                    collink[0].click() # error at here after click 
                except:
                    print "Unexpected error:", sys.exc_info()[0] 
                    numberofmore=0
    selenium.implicitly_wait(3)
    numberofpages=numberofpages+1
    #print("here5")
```

**Comment by lmtierney** [comment]
Going to need more of the stack trace and the actual error message

**Comment by hoyeunglee** [comment]
Traceback (most recent call last):
  File "C:\Users\martlee2\Downloads\autoweb.py", line 77, in <module>
    collink[0].click()
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webelement.py", line 77, in click
    self._execute(Command.CLICK_ELEMENT)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webelement.py", line 493, in _execute
    return self._parent.execute(command, params)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 249, in execute
    self.error_handler.check_response(response)
  File "C:\Python27\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 193, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: unknown error: Element is not clickable at point (385, 1901)
  (Session info: chrome=61.0.3163.100)
  (Driver info: chromedriver=2.29.461591 (62ebf098771772160f391d75e589dc567915b233),platform=Windows NT 10.0.14393 x86_64)

**Comment by lmtierney** [comment]
With chrome 61, they changed how the browser scrolled items into view. They offered a fix in chromedriver `2.32` so please update your chromedriver. If you're still having issues after upgrading, please raise a bug to the chromium project https://bugs.chromium.org/p/chromedriver/issues/list

**Comment by hoyeunglee** [comment]
thanks, it works.
